lib/std/os/windows/ws2_32.zig

```diff
@@ -7,10 +7,879 @@ usingnamespace @import("bits.zig");
+pub const IOC_WS2 = @as(u32, 134217728);
+pub const IOC_PROTOCOL = @as(u32, 268435456);
+pub const IOC_VENDOR = @as(u32, 402653184);
+pub const IPPROTO_IP = @as(u32, 0);
+pub const IPPORT_TCPMUX = @as(u32, 1);
+pub const IPPORT_ECHO = @as(u32, 7);
+pub const IPPORT_DISCARD = @as(u32, 9);
+pub const IPPORT_SYSTAT = @as(u32, 11);
+pub const IPPORT_DAYTIME = @as(u32, 13);
+pub const IPPORT_NETSTAT = @as(u32, 15);
+pub const IPPORT_QOTD = @as(u32, 17);
+pub const IPPORT_MSP = @as(u32, 18);
+pub const IPPORT_CHARGEN = @as(u32, 19);
+pub const IPPORT_FTP_DATA = @as(u32, 20);
+pub const IPPORT_FTP = @as(u32, 21);
+pub const IPPORT_TELNET = @as(u32, 23);
+pub const IPPORT_SMTP = @as(u32, 25);
+pub const IPPORT_TIMESERVER = @as(u32, 37);
+pub const IPPORT_NAMESERVER = @as(u32, 42);
+pub const IPPORT_WHOIS = @as(u32, 43);
+pub const IPPORT_MTP = @as(u32, 57);
+pub const IPPORT_TFTP = @as(u32, 69);
+pub const IPPORT_RJE = @as(u32, 77);
+pub const IPPORT_FINGER = @as(u32, 79);
+pub const IPPORT_TTYLINK = @as(u32, 87);
+pub const IPPORT_SUPDUP = @as(u32, 95);
+pub const IPPORT_POP3 = @as(u32, 110);
+pub const IPPORT_NTP = @as(u32, 123);
+pub const IPPORT_EPMAP = @as(u32, 135);
+pub const IPPORT_NETBIOS_NS = @as(u32, 137);
+pub const IPPORT_NETBIOS_DGM = @as(u32, 138);
+pub const IPPORT_NETBIOS_SSN = @as(u32, 139);
+pub const IPPORT_IMAP = @as(u32, 143);
+pub const IPPORT_SNMP = @as(u32, 161);
+pub const IPPORT_SNMP_TRAP = @as(u32, 162);
+pub const IPPORT_IMAP3 = @as(u32, 220);
+pub const IPPORT_LDAP = @as(u32, 389);
+pub const IPPORT_HTTPS = @as(u32, 443);
+pub const IPPORT_MICROSOFT_DS = @as(u32, 445);
+pub const IPPORT_EXECSERVER = @as(u32, 512);
+pub const IPPORT_LOGINSERVER = @as(u32, 513);
+pub const IPPORT_CMDSERVER = @as(u32, 514);
+pub const IPPORT_EFSSERVER = @as(u32, 520);
+pub const IPPORT_BIFFUDP = @as(u32, 512);
+pub const IPPORT_WHOSERVER = @as(u32, 513);
+pub const IPPORT_ROUTESERVER = @as(u32, 520);
+pub const IPPORT_RESERVED = @as(u32, 1024);
+pub const IPPORT_REGISTERED_MAX = @as(u32, 49151);
+pub const IPPORT_DYNAMIC_MIN = @as(u32, 49152);
+pub const IPPORT_DYNAMIC_MAX = @as(u32, 65535);
+pub const IN_CLASSA_NET = @as(u32, 4278190080);
+pub const IN_CLASSA_NSHIFT = @as(u32, 24);
+pub const IN_CLASSA_HOST = @as(u32, 16777215);
+pub const IN_CLASSA_MAX = @as(u32, 128);
+pub const IN_CLASSB_NET = @as(u32, 4294901760);
+pub const IN_CLASSB_NSHIFT = @as(u32, 16);
+pub const IN_CLASSB_HOST = @as(u32, 65535);
+pub const IN_CLASSB_MAX = @as(u32, 65536);
+pub const IN_CLASSC_NET = @as(u32, 4294967040);
+pub const IN_CLASSC_NSHIFT = @as(u32, 8);
+pub const IN_CLASSC_HOST = @as(u32, 255);
+pub const IN_CLASSD_NET = @as(u32, 4026531840);
+pub const IN_CLASSD_NSHIFT = @as(u32, 28);
+pub const IN_CLASSD_HOST = @as(u32, 268435455);
+pub const INADDR_LOOPBACK = @as(u32, 2130706433);
+pub const INADDR_NONE = @as(u32, 4294967295);
+pub const IOCPARM_MASK = @as(u32, 127);
+pub const IOC_VOID = @as(u32, 536870912);
+pub const IOC_OUT = @as(u32, 1073741824);
```

Comment:
Many of these are much more readable in hex form.
```suggestion
pub const IOC_OUT:u32 = 0x40000000;
```

Reply:
Agreed, though this is output directly from the auto-generated bindings from @marler8997 - unsure if the constants are intentionally in decimal form due to the generator, or if the constants provided by Microsoft are in hex form.

Reply:
cc @marler8997: would love to know if this is something that could be corrected by the bindings generator